doc/conf.py

```diff
@@ -82,35 +89,45 @@
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 
 # -- Options for HTML output ----------------------------------------------
-
-# see: https://sphinx-bootstrap-theme.readthedocs.io/en/latest/README.html
-# Clean up sidebar: Do not show "Source" link
 html_show_sourcelink = False
+html_copy_source = False
 
-html_theme = 'bootstrap'
-html_theme_path = sphinx_bootstrap_theme.get_html_theme_path()
-
+html_theme = 'pydata_sphinx_theme'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
+switcher_version_match = "dev" if "dev" in release else version
+
 html_theme_options = {
-    'navbar_title': 'autoreject',
-    'bootswatch_theme': "united",
-    'navbar_sidebarrel': False,  # no "previous / next" navigation
-    'navbar_pagenav': False,  # no "Page" navigation in sidebar
-    'bootstrap_version': '3',
-    'navbar_links': [
-        ("Examples", "auto_examples/index"),
-        ("Explanation", "explanation"),
-        ("FAQ", "faq"),
-        ("API", "api"),
-        ("What's new", "whats_new"),
-        ("GitHub", "https://github.com/autoreject/autoreject", True)
+    'icon_links': [
+        dict(name='GitHub',
+             url='https://github.com/autoreject/autoreject',
+             icon='fab fa-github-square'),
     ],
+    'icon_links_label': 'Quick Links',  # for screen reader
+    'use_edit_page_button': False,
+    'navigation_with_keys': False,
+    'show_toc_level': 1,
+    "navbar_end": ["theme-switcher", "version-switcher", "navbar-icon-links"],
+    "switcher": {
+        # "json_url": "https://mne.tools/dev/_static/versions.json",
+        "json_url": "https://raw.githubusercontent.com/sappelhoff/autoreject/pydata/sphinx/doc/_static/versions.json",  # noqa: E501
```

Comment:
This suggestion must be merged before merging this PR.

```suggestion
        "json_url": "https://raw.githubusercontent.com/autoreject/autoreject/pydata/sphinx/doc/_static/versions.json",  # noqa: E501
```